dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -34,6 +25,20 @@ class SentryStackTraceFactory {
         ? Chain.forTrace(stackTrace)
         : Chain.parse(stackTrace as String);
 
+    // if strip symbols are enabled, thats what we see:
+    // warning:  This VM has been configured to produce stack traces that violate the Dart standard.
+    // ***       *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
+    // unparsed  pid: 30930, tid: 30990, name 1.ui
+    // unparsed  build_id: '5346e01103ffeed44e97094ff7bfcc19'
```

Comment:
this would go to `debug_images`, but if we use the list from sentry-native and sentry-cocoa, it'd not be necessary